WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewController.swift

```diff
@@ -889,11 +889,6 @@ private extension SettingsViewController {
             comment: "Opens iOS's Device Settings for the app"
         )
 
-        static let launchWormHolyDebug = NSLocalizedString(
-            "Launch Wormholy Debug",
-            comment: "Opens an internal library called Wormholy. Not visible to users."
-        )
```

Comment:
The next time a release runs, we should expect all the "Launch Wormholy Debug" entries in the localization files to be removed.